### ISSUE TITLE
Datepicker buttons don't use light colour in dark theme

### ISSUE THREAD

**Issue by miaulalala** [nextcloud/calendar#4617]
### Steps to reproduce

Open datepicker in calendar event, or in when picking a recurrence end date
![image](https://user-images.githubusercontent.com/7427347/195606948-e05ba835-7889-43ae-a2aa-5a0a24c7d4a4.png)

### Expected behavior

The buttons should be white

### Actual behaviour

The buttons are dark grey

### Calendar app version

v4.0.0-rc.1

### CalDAV-clients used

Web

### Browser

Chrome

### Client operating system

Ubuntu

### Server operating system

c.nc

**Comment by SebastianKrupinski** [comment]
Closing this as no longer relevant, we use the native browser date/time picker now
